components/automate-ui/src/app/pages/job-add/job-add.component.ts

```diff
@@ -256,23 +254,24 @@ export class JobAddComponent implements OnDestroy {
     const ruleOpts = {
       dtstart: new Date(
         parseInt(start.datetime.year, 10),
-        parseInt(start.datetime.month, 10) - 1,
+        parseInt(start.datetime.month, 10),
         parseInt(start.datetime.date, 10),
-        this.hours12to24(parseInt(start.datetime.hour, 10), start.datetime.meridiem === 'PM'),
+        parseInt(start.datetime.hour, 10),
         parseInt(start.datetime.minute, 10)
```

Comment:
Similar to above, but for parsing now.  We are no longer using AM/PM and are using 24h format, so function is not needed.